File: task-03/Go/prime_number.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Print("Enter the limit: ")
	_, err := fmt.Scan(&n)
	if err != nil || n < 2 {
		fmt.Println("No Prime")
		return
	}

	fmt.Print("The prime numbers are:")
	for i := 2; i <= n; i++ {
		count := 0
		for j := 2; j*j <= i; j++ {
			if i%j == 0 {
				count = 1
				break
			}
		}
		if count == 0 {
			fmt.Printf(" %d", i)
		}
	}
	fmt.Println() // Print a newline to end the output
}

File: task-03/Javascript/prime_number.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter the limit: ', (answer) => {
  const n = parseInt(answer);

  if (isNaN(n) || n < 2) {
    console.log('No Prime');
    rl.close();
    return;
  }

  console.log('The prime numbers are:');
  for (let i = 2; i <= n; i++) {
    let count = 0;
    for (let j = 2; j * j <= i; j++) {
      if (i % j === 0) {
        count = 1;
        break;
      }
    }
    if (count === 0) {
      process.stdout.write(` ${i}`);
    }
  }
  console.log(); // Print a newline to end the output
  rl.close();
});

